nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+ * `EnableExpress(bool)`: Enables express mode, defaults to false.
+ * `EnableExpress(Func<string, bool>, string)`: Enables express mode when the given condition is true.
+ * `AutoDeleteOnIdle(TimeSpan)`: Auto deletes the entity if it hasn't been used for the given time frame.
+ * `EnablePartitioning(bool)`: Enables entity partitioning, defaults to false.
+ * `EnableBatchedOperations(bool)`: Enables server side batched operations, defaults to true.
+ * `RequiresDuplicateDetection(bool)`: Specifies whether the queue should perform duplicate detection, defaults to false.
+ * `DuplicateDetectionHistoryTimeWindow(TimeSpan)`: The time period in which duplicate detection should occur. 
+ * `SupportOrdering(bool)`: Best effort message ordering on the queue, defaults to false.
+ * `DescriptionFactory(Func<string, ReadOnlySettings, QueueDescription>)`: A factory method that allows to create a QueueDescription object from the Azure Service Bus SDK. Use this factory method to override any (future) setting that is not supported by the Queues API.
+
+### Topics
+ 
+The following settings are available to define how topics should be created.
+
+ * `MaxSizeInMegabytes(SizeInMegabytes)`: The size of the topic, in megabytes.
+ * `DefaultMessageTimeToLive(TimeSpan)`: The maximum age of a message, defaults to TimeSpan.MaxValue. 
+ * `AutoDeleteOnIdle(TimeSpan)`: Auto deletes the entity if it hasn't been used for the given time frame.
+ * `EnableBatchedOperations(bool)`: Enables server side batched operations, defaults to true.
+ * `EnableExpress(bool)`: Enables express mode, defaults to false.
+ * `EnableExpress(Func<string, bool>, bool)`: Enables express mode when the given condition is true.
+ * `EnableFilteringMessagesBeforePublishing(bool)`: Enables message filtering before they are published, defaults to false.
+ * `EnablePartitioning(bool)`: Enables entity partitioning, defaults to false.
+ * `RequiresDuplicateDetection(bool)`: Specifies whether the topic should perform duplicate detection, defaults to false.
+ * `DuplicateDetectionHistoryTimeWindow(TimeSpan)`: The time period in which duplicate detection should occur. 
+ * `SupportOrdering(bool)`: Best effort message ordering on the topic, defaults to false.
+ * `DescriptionFactory(Func<string, ReadOnlySettings, TopicDescription>)`: A factory method that allows to create a TopicDescription object from the Azure Service Bus SDK. Use this factory method to override any (future) setting that is not supported by the Topics API.
+
+### Subscriptions
+
+The following settings are available to define how subscriptions should be created.
+ 
+ * `DefaultMessageTimeToLive(TimeSpan)`: The maximum age of a message, defaults to TimeSpan.MaxValue. 
+ * `EnableBatchedOperations(bool)`: Enables server side batched operations, defaults to true.
+ * `EnableDeadLetteringOnFilterEvaluationExceptions(bool)`: Dead letters messages when a filter evaluation doesn't match, defaults to false.
```

Comment:
Is "dead letters" a real expression? Should it be "sends messages to a dead letter queue..." or something similar?